libraries/ms-drivers/inc/pca9539r_gpio_expander.h

```diff
@@ -1,8 +1,9 @@
 #pragma once
 // GPIO HAL interface for the PCA9539RPW/Q900J GPIO expander.
-// Requires I2C to be initialized.
+// Requires I2C , GPIO_IT to be initialized.
```

Comment:
We have a bunch of projects which use the PCA9539R driver already, but they don't all need to be updated to initialize GPIO interrupts. So instead of adding to the `Requires I2C to be initialized` line, add another comment below it and say that *if* you want to subscribe to interrupts, you must initialize gpio interrupts.